website/docs/reference/parsing.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "Project Parsing"
+description: "Read this guide to understand the project parsing configuration in dbt.Read this guide to understand the project parsing configuration in dbt."
```

Comment:
Looks like this description was added twice

Reply:
https://deploy-preview-3261--docs-getdbt-com.netlify.app/reference/commands/parse

content="Read this guide on how dbt's parse command can be used to parse your dbt project and write detailed timing information."